Sources/VenmoPayments/VenmoClient.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public class VenmoClient: NSObject {
```

Comment:
Nit we don't need ObjC support for PPCP

```suggestion
public class VenmoClient {
```